ucm2/Intel/sof-essx8336/sof-essx8336.conf

```diff
@@ -15,50 +15,3 @@ SectionUseCase."HiFi" {
 	File "/Intel/sof-essx8336/HiFi.conf"
 	Comment "Play and record HiFi quality Music"
 }
-
-# the kcontrols initial values, which will be set by `alsactl init`
-If.speaker {
-	Condition {
-		Type ControlExists
-		Control "DAC Playback Volume"
-	}
-	True.BootSequence [
-		cset "name='DAC Playback Volume' 60%"
-	]
-}
-
-If.headphone {
-	Condition {
-		Type ControlExists
-		Control "name='Headphone Playback Volume'"
-	}
-	True.BootSequence [
-		cset "name='Headphone Playback Volume' 60%"
-	]
-}
-
-If.dmic {
-	Condition {
-		Type String
-		Empty "${var:DeviceDmic}"
-	}
-	False.If.Dmic0 {
-		Condition {
-			Type ControlExists
-			Control "name='Dmic0 Capture Volume'"
-		}
-		True.BootSequence [
-			cset "name='Dmic0 Capture Volume' 70%"
-		]
-	}
-}
```

Comment:
this is not related to ES8336 but Intel DMIC support. not sure why you removed it?

Reply:
no particular reason. I'll drop this hunk.